docs/docfx-template/partials/class.tmpl.partial

```diff
@@ -0,0 +1,238 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
```

Comment:
Good question. I *think* it's probably appropriate to keep it. (It doesn't come out in the generated docs, btw.)